notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<ol><li>I would start by saying "Prophet is Meta's forecasting model."</li><li>The location of the paper link ("See the <a href="https://peerj.com/preprints/3190/" target="_blank">paper</a> for details.") makes it seem like the paper is a comparison between BQML ARIMA+ and Prophet. But it's a paper about Prophet only, right?</li><li>In our ARIMA notebook / documentation, we make an important distinction between univariate and multivariate models. In which category does Prophet fall under?</li><li>Since Tabular Workflows provides support for multiple time series, should we state in the introduction that Prophet is a model for a single time series?</li><li>I feel like we should at least have a reference to each of these features somewhere in the body of the notebook. I see you've added references to Dataflow, that's good. Can we do the same for data splitting, windowing strategies, hyperparameter tuning?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Replaced the Overview section with what you wrote in the documentation page.